bpf_test.go

```diff
@@ -234,6 +239,26 @@ func checkSocketFilters(t *testing.T, b *elf.Module) {
 	}
 }
 
+func checkPinCustomConfig(t *testing.T, b *elf.Module) {
+	// NOTE: this test doesn't really work kernel 4.9 or older,
+	// because /sys/fs/bpf/fw doesn't exist at all.
```

Comment:
What is the name of the feature that is missing before 4.9?

https://github.com/iovisor/bcc/blob/master/docs/kernel-versions.md